src/modules/temporaryWindowInjection.ts

```diff
@@ -310,9 +318,13 @@ Object.assign(<any>window, {
     EffectEngineRunner,
     ItemHandler,
     CaughtIndicatingItem,
+    EggItem,
+    FluteItem,
+    MegaStoneItem,
     PokeballItem,
+    Vitamin,
+    VitaminController,
     RoamingPokemonList,
     DataPokemon,
-    MegaStoneItem,
```

Comment:
Why was this removed? I think it is needed in DungeonRunner to check if an item is an instanceof MegaStoneItem

Reply:
Was moved up a bit, to be with the other items.